src/scipp/utils/get.py

```diff
@@ -5,18 +5,20 @@
 
 def get(x, key, default=None):
     """
-    Get the value associated with the provided key from a `dict`-like object
+    Get the value associated with the provided key from a `dict`-like object.
 
-    :param x: dictlike
-        Any object with `__contains__()` or `__iter__()` and `__getitem__()`
-        methods which accept `key`
+    :param x:
+        Any object with a `__getitem__()` method which accept `key`.
+        Otherwise, a TypeError is thrown.
```

Comment:
Since this function is almost exclusively used as a method, shouldn't we skip documenting the "self" param?

Reply:
Yes, remove it.